compiler/src/dmd/dsymbolsem.d

```diff
@@ -508,10 +508,10 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         {
             if (inferred)
             {
-                dsym.error("type `%s` is inferred from initializer `%s`, and variables cannot be of type `void`", dsym.type.toChars(), dsym._init.toChars());
+                dsym.error("- type `%s` is inferred from initializer `%s`, and variables cannot be of type `void`", dsym.type.toChars(), dsym._init.toChars());
```

Comment:
Is `-` really needed?  What about if the message used `:`?

Not objecting, just considering all punctuation marks. :-)

Reply:
I used a colon first but it gets formatted with a space before the colon, which looked a bit odd. So I think a hyphen looks better.